src/components/navigation.js

```diff
@@ -3,22 +3,24 @@ import { Link } from "gatsby"
 import styled from "styled-components"
 
 const Navigation = () => (
-  <nav>
+  <nav
+    style={{
+      width: "50%",
+    }}
+  >
     <LinksWrapper>
-      <div id="linksInnerWrapper">
-        <li>
-          <Link to="/">Home</Link>
-        </li>
-        <li>
-          <Link to="/">Why us?</Link>
-        </li>
-        <li>
-          <Link to="/">How it works</Link>
-        </li>
-        <li>
-          <Link to="/">Foddies</Link>
-        </li>
-      </div>
+      <li>
+        <Link to="/">Home</Link>
```

Comment:
estos no van a ser `<Link to`, van a ser `a` y en onclick van a animar el scrollbar